### PR TITLE
#2304 In Semantic Browser, add "Involved Activities" category for "Operational Capability"

### PR DESCRIPTION
**Pull request by nlepine**

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 1]
```diff
@@ -1113,6 +1113,22 @@
                class="org.polarsys.capella.core.data.oa.OperationalActivity">
          </availableForType>
       </category>
+      <category
+            id="org.polarsys.capella.core.semantic.queries.OperationalActivityInvolvingCapabilities"
+            isTopLevel="true"
+            name="Involving Capabilities">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:1119]: Involving Operational Capabilities (ie like https://github.com/eclipse/capella/blob/71d049a46b03c4996bd1fa38fb1b074f8c1be4e3/core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml#L4445)

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 2]
```diff
@@ -6330,6 +6346,22 @@
             </basicQuery>
          </categoryQuery>
       </category>
+      <category
+            id="org.polarsys.capella.core.semantic.queries.AbstractCapabilityInvolvedFunctionsOA"
+            isTopLevel="true"
+            name="Involved Activities">
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:6352]: Involving Operational Activities

**Review by pdulth — CHANGES_REQUESTED** [review]

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml (hunk 1 above)
Comment (nlepine) [reply core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:1119]: ok

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml (hunk 2 above)
Comment (nlepine) [reply core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:6352]: ok

File: core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?eclipse version="3.2"?>
 <!--
-  Copyright (c) 2018, 2020 THALES GLOBAL SERVICES.
+  Copyright (c) 2018, 2022 THALES GLOBAL SERVICES.
```
Comment (mPorhel) [inline core/plugins/org.polarsys.capella.core.semantic.queries/plugin.xml:4]: Copyright (c) 2018, 2022 THALES GLOBAL SERVICES and others.

+ contributors - summary/scope of the change/ticket id

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/OperationalActivityInvolvingCapabilities.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
```
Comment (mPorhel) [inline core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/OperationalActivityInvolvingCapabilities.java:2]: Copyright (c) 2022 Obeo.

contributors - summary/scope of the change/ticket id

File: tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/OperationalActivity_InvolvingCapabilities.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
```
Comment (mPorhel) [inline tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/OperationalActivity_InvolvingCapabilities.java:2]: Idem here, copyright must be changed before merge.

File: tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/OperationalCapability_InvolvedActivities.java [hunk 6]
```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
```
Comment (mPorhel) [inline tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/OperationalCapability_InvolvedActivities.java:2]: Idem here, copyright must be changed before merge.

File: tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testsuites/SemanticQueriesTestSuite.java [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2021 THALES GLOBAL SERVICES.
+ * Copyright (c) 2018, 2022 THALES GLOBAL SERVICES.
```
Comment (mPorhel) [inline tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testsuites/SemanticQueriesTestSuite.java:2]: Idem here, copyright must be changed before merge.

**Review by mPorhel — CHANGES_REQUESTED** [review]
@nlepine please review the copyrights headers.

**Comment by pdulth** [comment]
for me looks good. @mPorhel ?

**Comment by mPorhel** [comment]
@pdulth Sounds good for me it suits Capella rules.

I would just have had the Signed-off-by: Nathalie Lépine <nathalie.lepine@obeo.fr>

**Pull request merged**